tests/unit/helpers/questionnaires.test.js

```diff
@@ -311,7 +335,10 @@ describe('getUserQuestionnaires', () => {
     const course = {
       _id: new ObjectID(),
       format: 'blended',
-      slots: [{ startDate: new Date('2021-04-20T09:00:00'), endDate: new Date('2021-04-20T11:00:00') }],
+      slots: [
```

Comment:
Pour le nom de ce test je mettrais 'should return an empty array if first slot is passed but last slot isn\'t', qu'en penses-tu ?

Reply:
done